binding.gyp

```diff
@@ -25,13 +25,13 @@
       'target_name': 'jstp',
       'sources': [
         'src/node_bindings.cc',
-        'src/jsrs_parser.cc',
+        'src/json5_parser.cc',
         'src/packet_parser.cc',
         'src/unicode_utils.cc'
       ],
       'conditions': [
         ['not jstp_use_short_unicode_tables', {
-          'defines': ['_JSRS_USE_FULL_TABLES_']
+          'defines': ['_JSTP_USE_FULL_TABLES_']
```

Comment:
`jstp_use_short_unicode_tables` is the name of the environment variable a user may define while installing the package, and they deal with the package as a whole, thus the name. The name of the preprocessor variable, to the contrary, is internal and applies to the JSON5 subsystem only.